README.md

```diff
@@ -3,35 +3,68 @@ Repository to experiment with serverless framework and automation.
 
 This project uses serverless framework and attempts to create a serverless environment that could be used to automate vulnerability assessment tasks with multiple ingestion points, such as on-demand submission of a host via a REST API, regular scanning of a list of hosts and proactive scanning of hosts appearing in Certificate Transparency logs.
 
-This is under development with more features being added as different branches. `master` branch currently only supports:
-- Scheduled HTTP Observatory scans (runs every 5 minutes)
-- HTTP Observatory scan of a host manually submitted to an SQS queue
-Ingestion points for both is a hard-coded list of hosts (for PoC purposes).
+This is under development with more features being added as different branches. The current branch supports:
+- Addition of a target to the scan queue for port scan by an API endpoint (`/ondemand/portscan`)
+- Addition of a target to the scan queue for Observatory scan by an API endpoint (`/ondemand`)
```

Comment:
should this instead be /ondeman/[obs reference]?

Reply:
The documentation is correct, however it makes sense to change that endpoint and update the documentation. 
Will add it to master soon as it's a small change.